### PR TITLE
Loris form updates for Instruments

### PR DESCRIPTION
**Pull request by jstirling91**
Removes unused code for instrument preview, adds functionality for uploading files using LorisForm and fixes bug with add elements with null names

### CODE REVIEW

File: smarty/templates/instrument_html.tpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	     border-top: none; 
 	 }
 </style>
-<form {$form.attributes}>
+<form method="post" name="test_fosrm" id="test_form" {$form.enctype}>
```
Comment (gluneau) [inline smarty/templates/instrument_html.tpl]: Should this name be "test_forms" ?

**Comment by stellarxo** [comment]
@gluneau can you please let me know when this has been put on IBIS staging?

**Comment by taracampbell** [comment]
@jstirling91 I have gone through all the modules we regularly use in CCNA and everything seems to be functioning normally. I will do instruments this afternoon... I have a few questions about the setup.

**Comment by stellarxo** [comment]
vendor/bin/phpcs --standard=docs/LorisCS.xml php/libraries php/exceptions

**Comment by alisterdev** [comment]
Seems that `selectmultiple` from a .linst file shows as a simple dropdown now. Did something change/wasn't implemented yet?

File: php/libraries/NDB_BVL_Instrument_LINST.class.inc [hunk 2]
```diff
@@ -49,7 +49,7 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
         $this->commentID = $commentID;
         $this->page      = $page;
 
-        $this->form = new \HTML_Quickform('test_form');
+        $this->form = new \LorisForm('test_form');
```
Comment (alisterdev) [inline php/libraries/NDB_BVL_Instrument_LINST.class.inc]: I believe that [line 426](https://github.com/jstirling91/Loris/blob/19d893e5ea175f82f18b75b8e022b8b6de6a9cf8/php/libraries/NDB_BVL_Instrument_LINST.class.inc#L426) in this file also needs to be updated

**Comment by driusan** [comment]
@jstirling91 Can you make sure this passes Travis so that it can be merged now that people have tested it?

**Comment by davidblader** [comment]
i've encountered a bug with time elements. currently, fields which require `hours : minutes` format take a `MM/DD/YYYY` format on firefox and a `YYYY-MM-DD` format on chrome

**Comment by alisterdev** [comment]
@davidblader I believe it is fixed on the 16.1 along with other datepicker bugs, but was not yet pulled to 17.0

**Comment by jstirling91** [comment]
@davidblader can you provide an example where this is happening so I can test it myself?

**Comment by davidblader** [comment]
@jstirling91 Biosample Collection instrument on CCNA, in the clinical assessment test battery

**Pull request merged**